src/content/reference/platform-api/cluster-apps/default-apps-aws.md

```diff
@@ -0,0 +1,2920 @@
+---
```

Comment:
Merged into cluster-aws and therefore not relevant anymore. Same for the others (except EKS but I'd just rip out `default-apps-*` overall).